persistence/persistence-mybatis-test/src/test/resources/testSql.sql

```diff
@@ -17,6 +17,9 @@ DROP TABLE IF EXISTS "genetic_profile";
 DROP TABLE IF EXISTS "gene";
 DROP TABLE IF EXISTS "cancer_study";
 DROP TABLE IF EXISTS "type_of_cancer";
+DROP TABLE IF EXISTS "gene_panel";
+DROP TABLE IF EXISTS "gene_panel_list";
```

Comment:
gene_panel_list has gene_panel and gene as parent tables in foreign key constraints. It must be dropped before any parent table is dropped because of this relationship

gene_panel_profile_map has gene_panel and sample and genetic_profile as parent tables in foreign key constraints. It must be dropped before any parent table is dropped because of this relationship

Move the DROP gene_panel_list line to a position above the Drop gene line
Move the DROP gene_panel_profile_map line to a position above the Drop sample line